frontend/src/app/modules/settings/video-conferencing/video-conferencing.component.ts

```diff
@@ -23,7 +23,10 @@ export class VideoConferencingComponent extends BaseComponent {
         private activatedRoute: ActivatedRoute,
     ) {
         super();
-        spinnerService.show();
+    }
+
+    ngOnInit(): void {
+        this.spinnerService.show();
```

Comment:
You need to wait for all observables and only after that hide spinner

Reply:
Fixed